src/components/Search/styles.scss

```diff
@@ -0,0 +1,36 @@
+@import '../../styles/main.scss';
+.search-container {
+    float: right;
+    height: 100vh;
+    overflow: auto;
+    background-color: white;
+    @include flex-center();
+    @include respond-above(md) {
+      width: rem(1140px);
+    }
+  }
+
+@mixin button-bg($bg) {
+    background: $bg;
+    &:hover {
+      background:darken($bg,8%);
+      transition: all 0.3s ease;
+    }
+    &:active {
+      background:darken($bg,25%);
+    } 
+  }
+
+  .btn {
+    color:white;
+    text-decoration:none;
+    padding:5px 10px;
+    border-radius:3px;
+    
+   }
+   
+   .btn-search {
+      @include button-bg(#47a1f5);
+   }
```

Comment:
Since we're using `scss`, we should follow its convention. Also rename the components to match `BEM`.
Example solution: https://github.com/Cobda/cobda-web/pull/4/files#r562792041
Naming conventions: https://github.com/Cobda/cobda-web/pull/4/files#r562792041
`scss`: https://css-tricks.com/sass-style-guide/